tests/test_dictdiffer.py

```diff
@@ -54,6 +54,14 @@ def test_change(self):
         diffed = next(diff(first, second))
         assert ('change', 'a', ('b', None)) == diffed
 
+        first = {'a': 10.0}
+        second = {'a': 10.5}
+        diffed = list(diff(first, second, tolerance=0.1))
+        assert [] == diffed
+
+        diffed = next(diff(first, second, tolerance=0.01))
+        assert ('change', 'a', (10.0, 10.5)) == diffed
+
```

Comment:
@jirikuncar I pushed a new branch with a new parameter `convert` to allow string conversion to numerical value for comparison.